code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -280,6 +280,8 @@
 
 	if(user)
 		var/datum/hud/HUDdatum = global.HUDdatums[user.defaultHUD]
+		if (!istype(HUDdatum) || !HUDdatum.ConteinerData || !HUDdatum.ConteinerData.len)
```

Comment:
robots being able to interact with storages to some degree was cauysing this proc to runtime